channel.json

```diff
@@ -124,6 +124,7 @@
 		"https://raw.githubusercontent.com/xgenvn/InputHelper/master/packages.json",
 		"https://raw.githubusercontent.com/zfkun/sublime-kissy-snippets/master/packages.json",
 		"https://sublimerge.com/packages.json",
-		"https://tinkerwell.app/plugins/sublime/packages.json"
+		"https://tinkerwell.app/plugins/sublime/packages.json",
+		"https://raw.githubusercontent.com/Exafunction/codeium-sublime/main/packages.json"
```

Comment:
Please also put this line in lexicographical order. I'll take a look at the new package's contents at a later time.

Reply:
Done